### PR TITLE
Change SSIM code example to use real MSE

### PR DESCRIPTION
**Pull request by jjerphan**
## Description

Quick fix for #4743

## Checklist

- [x] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [x] Gallery example in `./doc/examples` (new features only)
- [x] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [x] Unit tests
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,7 +1,6 @@
 include setup*.py
 include MANIFEST.in
-include README.md
-include LICENSE.txt
+include *.md
```
Comment (jni) [inline MANIFEST.in:3]: Why this change?

File: doc/examples/transform/plot_ssim.py [hunk 2]
```diff
@@ -26,33 +26,29 @@
 
 from skimage import data, img_as_float
 from skimage.metrics import structural_similarity as ssim
-
+from skimage.metrics import mean_squared_error
```
Comment (jni) [inline doc/examples/transform/plot_ssim.py:29]:
```suggestion
from skimage.metrics import mean_squared_error

```

File: MANIFEST.in (hunk 1 above)
Comment (jjerphan) [reply MANIFEST.in:3]: Hum, I think I or my IDE did something weird.
